wiki/Tournaments/UKCC/UKCC7/en.md

```diff
@@ -0,0 +1,580 @@
+- Hard Rock
+    - [Breaking Benjamin - The Diary of Jane [Extreme]](https://osu.ppy.sh/beatmaps/1001921)
+    - [Eagle - S!ck [Extra]](https://osu.ppy.sh/beatmaps/341285)
+    - [An - Encryption [Kloyd's Extra]](https://osu.ppy.sh/beatmaps/258384)
+- Double Time
+    - [Nanamori-chu \* Goraku-bu - Precious Friends [Insane]](https://osu.ppy.sh/beatmaps/420131)
+    - [Chata - Present [Insane]](https://osu.ppy.sh/beatmaps/1761649)
+    - [Mago de Oz - Xanandra [Sieg's Hard]](https://osu.ppy.sh/beatmaps/249032)
+- Free Mod
+    - [nano - Silence [Insane]](https://osu.ppy.sh/beatmaps/555116)
+    - [Streetlight Manifesto - A Moment of Violence [Extra]](https://osu.ppy.sh/beatmaps/1035104)
+    - [onoken - Felys [DJPop's Insane]](https://osu.ppy.sh/beatmaps/160795)
+- Tiebreaker
+    - [FELT - Lies in Reality [Albatross]](https://osu.ppy.sh/beatmaps/1205981)
+
+### Qualifying Round 2
+
+**[Download the map pack here! (86.8MB)](https://mega.nz/#!f3ASAA7T!umH3bN99cnuYe1ZFsOiXauUBFQRrW0n-_Zr4V8gE9ZA)**
+
+- Nomod
+    - [96neko - E? Aa, sou. [Fabricated]](https://osu.ppy.sh/beatmaps/1914227)
+    - [cYsmix - Peer Gynt [Expert]](https://osu.ppy.sh/beatmaps/1691360)
+    - [Reid Speed & Ahee - Overtaken (feat. Heartwurkz) [Tonight is All Mine]](https://osu.ppy.sh/beatmaps/1877730)
+- Hidden
+    - [FLOOR LEGENDS -KAC 2012- - KAC 2012 ULTIMATE MEDLEY -HISTORIA SOUND VOLTEX- [N a s y a's INFINITE]](https://osu.ppy.sh/beatmaps/682020)
+    - [Bilibili Douga - Night of Feifan Brother [Feifan Bro]](https://osu.ppy.sh/beatmaps/139221)
+- Hard Rock
+    - [nano - No pain, No game [Insane]](https://osu.ppy.sh/beatmaps/185761)
+    - [iru1919 - Tenko [Insane]](https://osu.ppy.sh/beatmaps/1759876)
+- Double Time
+    - [u's - START:DASH!! [Insane]](https://osu.ppy.sh/beatmaps/264252)
+    - [Hana - Gekka Jasmine (2014ver.) [Gyro's Light Insane]](https://osu.ppy.sh/beatmaps/1035068)
+
+*Maps were played in the order: NM3, HR2, DT2, HD2, HR1, NM2, DT1, NM1, HD1*
+
+
+### Qualifying Round 1
+
+**[Download the map pack here! (46.7MB)](https://mega.nz/#!T6RjDY7S!0PbquOystkS4v2-byiBC0b_KMEZA8lECFhMqjYKDm_A)**
+
+- Nomod
+    - [ALiCE'S EMOTiON - Cold Rain [Lunatic]](https://osu.ppy.sh/beatmaps/482650)
+    - [sakuzyo - VALLISTA [Another]](https://osu.ppy.sh/beatmaps/127313)
+    - [U - Ha-tenya? [Insane]](https://osu.ppy.sh/beatmaps/120080)
+    - [YUC'e - Sengoku HOP [apple's Insane]](https://osu.ppy.sh/beatmaps/1489274)
+- Hidden
+    - [Iguchi Yuka - RE-ILLUSION [Collab Insane]](https://osu.ppy.sh/beatmaps/1856697)
+- Hard Rock
+    - [Meramipop - algorhythm [Insane]](https://osu.ppy.sh/beatmaps/136571)
+- Double Time
+    - [T.M.Revolution x Mizuki Nana - Preserved Roses [Collab Hard]](https://osu.ppy.sh/beatmaps/522135)
+
+## Schedules / Match Results
+
+![Schedules](img/schedule_no_number.png)
+
+### Losers Final + Grand Final
+
+| Wednesday, 2019-10-23 | | | | |
```

Comment:
We had a new format recently. Please follow example below and apply for every table in match results.
```
Wednesday, 23 October 2019

| Player 1 |  |  | Player 2 | Match link |
| --: | :-: | :-: | :-- | :-- |
| **Bubbleman** ![][flag_GB] | **7** | 4 | ![][flag_GB] AJT | [#1](link) |
```
Also the player 2's structure is flag first then the username